recipes/openms-meta/build.sh

```
@@ -10,9 +10,32 @@ export LIBRARY_PATH=${PREFIX}/lib
+
+# Save current directory
+CURRENT_DIR=$(pwd)
+
+# Go to source directory
+cd $SRC_DIR
+
+if command -v tree &>/dev/null; then
+    tree -L 2 # Show only 2 levels deep to avoid excessive output
+fi
+
+cd OpenMS
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)